DCPvsOID/DCPvsOID.md

```diff
@@ -26,30 +26,30 @@ The Digital Product Passport (DPP)  is a digital record that contains essential
 The European Commission has mandated DPPs under the framework of the Ecodesign for Sustainable Products Regulation (ESPR). This regulation, adopted as part of the European Green Deal, aims to ensure products are designed with sustainability in mind and include data that supports their reuse, recycling, and responsible disposal. The implementation of DPPs will become mandatory for certain product categories starting in 2027, with a phased rollout across industries.
 
 
-## How Decentralized Identities and Verifiable Credentials enable the creation of trustworthiness in dataspaces
+## How Decentralized Identifiers and Verifiable Credentials enable the creation of trustworthiness in dataspaces
 
 ### Scenario
 Two parties, a component manufacturer (Company A) and a systems integrator (Company B), are collaborating within Manufacturing-X to share data regarding a digital product passport for a machine component. The DPP must include detailed information about the component’s materials, environmental impact, and compliance with safety standards.
 
-### Step 1: Establishing Decentralized Identities
-Decentralized identities form the backbone of trust in this ecosystem. Company A and Company B, each possess their own decentralized identifiers (DIDs), which are unique representations of their identities within the dataspace. These DIDs are coupled with verifiable credentials issued by trusted entities, such as regulatory authorities or certification bodies.
+### Step 1: Establishing Decentralized Identifiers
+Decentralized identifiers for policies, claims and evidence for the participating entities, materials, products, production site, ... form the backbone of trust in this ecosystem. Company A and Company B, each possess their own decentralized identifiers (DIDs), which are representations of their unique identities within the dataspace. These DIDs are coupled with verifiable credentials issued by trusted entities, such as regulatory authorities or certification bodies.
 
-For example, Company A might possess credentials verifying its compliance with ISO 14001 environmental management standards, while Company B holds credentials confirming its adherence to industry specific safety protocols.
+For example, using the Eclipse Conformity Assessment Profile, Company A might possess credentials verifying its compliance with ISO 14001 environmental management standards, while Company B holds credentials confirming its adherence to industry specific safety protocols.
```

Comment:
This sentence is barely readable, please reformulate. The added half-sentence should be a dedicated paragraph to explain the relation between both.